src/app/meshes/routes.ts

```diff
@@ -38,7 +38,7 @@ export const routes = (
               path: '',
               name: 'mesh-tabs-view',
```

Comment:
Should we make this consistent while we are here I think looking at the rest this should be `mesh-detail-tabs-view`, could also be done separately 🤷

Reply:
Done.